# Norwegian Nynorsk translation for update-manager
# Copyright (c) (c) 2006 Canonical Ltd, and Rosetta Contributors 2006
# This file is distributed under the same license as the update-manager package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
#: ../DistUpgrade/DistUpgradeController.py:1076
msgid ""
msgstr ""
"Project-Id-Version: update-manager\n"
"Report-Msgid-Bugs-To: sebastian.heinlein@web.de\n"
"POT-Creation-Date: 2020-06-24 15:42-0700\n"
"PO-Revision-Date: 2013-05-22 11:00+0000\n"
"Last-Translator: Åsmund Skjæveland <aasmunds@ulrik.uio.no>\n"
"Language-Team: Norwegian Nynorsk <nn@li.org>\n"
"Language: nn\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Launchpad-Export-Date: 2019-11-15 04:47+0000\n"
"X-Generator: Launchpad (build c597c3229eb023b1e626162d5947141bf7befb13)\n"
"X-Language: nn_NO\n"
"X-Source-Language: C\n"

#. TRANSLATORS: %s is a country
#: ../DistUpgrade/distro.py:215 ../DistUpgrade/distro.py:446
#, python-format
msgid "Server for %s"
msgstr "Tenar for %s"

#. More than one server is used. Since we don't handle this case
#. in the user interface we set "custom servers" to true and
#. append a list of all used servers
#: ../DistUpgrade/distro.py:233 ../DistUpgrade/distro.py:239
#: ../DistUpgrade/distro.py:255
msgid "Main server"
msgstr "Hovudtenar"

#: ../DistUpgrade/distro.py:259
msgid "Custom servers"
msgstr "Eigendefinerte tenarar"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:151
msgid "Could not calculate sources.list entry"
msgstr "Klarte ikkje å rekna ut oppføringa i sources.list"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:260
msgid ""
"Unable to locate any package files, perhaps this is not a Ubuntu Disc or the "
"wrong architecture?"
msgstr ""
"Klarte ikkje å finna nokon pakkefiler. Kanskje dette ikkje er ein Ubuntu-"
"disk eller rett arkitektur?"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:310
msgid "Failed to add the CD"
msgstr "Klarte ikkje å leggja til CD-en"

#: ../DistUpgrade/DistUpgradeAptCdrom.py:311
#, python-format
msgid ""
"There was a error adding the CD, the upgrade will abort. Please report this "
"as a bug if this is a valid Ubuntu CD.\n"
"\n"
"The error message was:\n"
"'%s'"
msgstr ""
"Det oppstod ein feil under lesing av CD-en, oppgraderinga vert avbroten. "
"Dette lyt rapporterast som ein feil, dersom dette er ein gyldig Ubuntu-CD.\n"
"\n"
"Feilmeldinga var:\n"
"\"%s\""

#: ../DistUpgrade/DistUpgradeCache.py:180
msgid "Remove package in bad state"
msgid_plural "Remove packages in bad state"
msgstr[0] "Fjern dårleg pakke"
msgstr[1] "Fjern dårlege pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:183
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Do you want to remove this package now "
"to continue?"
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archives can be found for them. Do you want to remove these packages "
"now to continue?"
msgstr[0] ""
"Pakken '%s' er øydelagd, og lyt leggjast inn på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Vil du fjerna pakken no og halda fram?"
msgstr[1] ""
"Pakkene '%s' er ødelagde, og lyt leggjast inn på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Vil du fjerna pakkene no og halda fram?"

#. FIXME: not ideal error message, but we just reuse a
#. existing one here to avoid a new string
#: ../DistUpgrade/DistUpgradeCache.py:299
msgid "The server may be overloaded"
msgstr "Tenaren kan vera oppteken"

#: ../DistUpgrade/DistUpgradeCache.py:412
msgid "Broken packages"
msgstr "Skadde pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:413
msgid ""
"Your system contains broken packages that couldn't be fixed with this "
"software. Please fix them first using synaptic or apt-get before proceeding."
msgstr ""
"Systemet ditt inneheld øydelagde pakkar som ikkje kunne reparerast med denne "
"programvara. Reparer dei med synaptic eller apt-get før du held fram."

#: ../DistUpgrade/DistUpgradeCache.py:649
#, fuzzy, python-format
msgid "Broken packages after upgrade: %s"
msgstr "Skadde pakkar"

#. the most likely problem is the 3rd party pkgs so don't address
#. foreignPkgs and devRelease being True
#: ../DistUpgrade/DistUpgradeCache.py:658
msgid ""
"An unresolvable problem occurred while calculating the upgrade.\n"
"\n"
" "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:661
msgid ""
"This was likely caused by:\n"
" * Unofficial software packages not provided by Ubuntu\n"
"Please use the tool 'ppa-purge' from the ppa-purge \n"
"package to remove software from a Launchpad PPA and \n"
"try the upgrade again.\n"
"\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:669
msgid ""
"This was caused by:\n"
" * Upgrading to a pre-release version of Ubuntu\n"
"This is most likely a transient problem, \n"
"please try again later.\n"
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:676
msgid "This is most likely a transient problem, please try again later."
msgstr "Dette er truleg berre eit kortvarig problem. Prøv om att seinare."

#: ../DistUpgrade/DistUpgradeCache.py:679
msgid ""
"If none of this applies, then please report this bug using the command "
"'ubuntu-bug ubuntu-release-upgrader-core' in a terminal. "
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:681
msgid ""
"If you want to investigate this yourself the log files in '/var/log/dist-"
"upgrade' will contain details about the upgrade. Specifically, look at 'main."
"log' and 'apt.log'."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:687
msgid "Could not calculate the upgrade"
msgstr "Klarte ikkje å førebu oppgraderinga"

#: ../DistUpgrade/DistUpgradeCache.py:746
msgid "Error authenticating some packages"
msgstr "Klarte ikkje å stadfesta identiteten åt somme pakkar"

#: ../DistUpgrade/DistUpgradeCache.py:747
msgid ""
"It was not possible to authenticate some packages. This may be a transient "
"network problem. You may want to try again later. See below for a list of "
"unauthenticated packages."
msgstr ""
"Somme pakkar kunne ikkje verta godkjende. Dette kan vera eit kortvarig "
"nettverksproblem, så du bør prøve om att seinare. Sjå under for lista over "
"pakkar som ikkje kunne godkjennast."

#: ../DistUpgrade/DistUpgradeCache.py:768
#, python-format
msgid ""
"The package '%s' is marked for removal but it is in the removal blacklist."
msgstr "Pakken '%s' er merkt for fjerning, men er i svartelista for fjerning."

#: ../DistUpgrade/DistUpgradeCache.py:774
#, python-format
msgid "The essential package '%s' is marked for removal."
msgstr "Den nødvendige pakken «%s» er merkt for fjerning."

#: ../DistUpgrade/DistUpgradeCache.py:783
#, python-format
msgid "Trying to install blacklisted version '%s'"
msgstr "Freistar å installera den svartelista versjon '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:905
#, python-format
msgid "Can't install '%s'"
msgstr "Kan ikkje installera '%s'"

#: ../DistUpgrade/DistUpgradeCache.py:906
msgid ""
"It was impossible to install a required package. Please report this as a bug "
"using 'ubuntu-bug ubuntu-release-upgrader-core' in a terminal."
msgstr ""

#: ../DistUpgrade/DistUpgradeCache.py:917
msgid "Can't guess meta-package"
msgstr "Klarte ikkje å gjetta på meta-pakke"

#: ../DistUpgrade/DistUpgradeCache.py:918
#, python-format
msgid ""
"Your system does not contain a %s or %s package and it was not possible to "
"detect which version of Ubuntu you are running.\n"
" Please install one of the packages above first using synaptic or apt-get "
"before proceeding."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:113
msgid "Reading cache"
msgstr "Les mellomlager"

#: ../DistUpgrade/DistUpgradeController.py:226
msgid "Unable to get exclusive lock"
msgstr "Klarte ikkje å få einerett"

#: ../DistUpgrade/DistUpgradeController.py:227
msgid ""
"This usually means that another package management application (like apt-get "
"or aptitude) already running. Please close that application first."
msgstr ""
"Dette tyder vanlegvis på at eit anna pakkehandsamingsprogram (slik som apt-"
"get eller aptitude) køyrer. Avslutt det programmet først."

#: ../DistUpgrade/DistUpgradeController.py:285
msgid "Upgrading over remote connection not supported"
msgstr "Å oppgradera over fjerntilkobling er ikkje støtta"

#: ../DistUpgrade/DistUpgradeController.py:286
msgid ""
"You are running the upgrade over a remote ssh connection with a frontend "
"that does not support this. Please try a text mode upgrade with 'do-release-"
"upgrade'.\n"
"\n"
"The upgrade will abort now. Please try without ssh."
msgstr ""
"Du køyrer oppgraderinga over ei ssh-tilkopling med grensesnitt som ikkje "
"støttar dette. Freist ei oppgradering i tekstmodus med 'do-release-"
"upgrade'.\n"
"\n"
"Oppgraderinga vil no avbrytast. Prøv utan ssh."

#: ../DistUpgrade/DistUpgradeController.py:300
msgid "Continue running under SSH?"
msgstr "Forsett køyringa under SSH?"

#: ../DistUpgrade/DistUpgradeController.py:301
#, python-format
msgid ""
"This session appears to be running under ssh. It is not recommended to "
"perform a upgrade over ssh currently because in case of failure it is harder "
"to recover.\n"
"\n"
"If you continue, an additional ssh daemon will be started at port '%s'.\n"
"Do you want to continue?"
msgstr ""
"Denne økta køyrer over ssh. Det er ikkje tilrådd å oppgradera på denne "
"måten, sia systemet vanskeleg let seg retta opp att ved feil.\n"
"\n"
"Viss du held fram, vil ei ekstra ssh-teneste bli starta på port '%s'.\n"
"Ønskjer du å halda fram?"

#: ../DistUpgrade/DistUpgradeController.py:315
msgid "Starting additional sshd"
msgstr "Startar endå ein sshd"

#: ../DistUpgrade/DistUpgradeController.py:316
#, python-format
msgid ""
"To make recovery in case of failure easier, an additional sshd will be "
"started on port '%s'. If anything goes wrong with the running ssh you can "
"still connect to the additional one.\n"
msgstr ""
"For å gjera gjenoppretting etter ein systemfeil lettare, vil ein ekstra sshd "
"verta starta på port '%s'. Dersom noko går gale med ssh-en som køyrer, kan "
"du framleis kopla til den nye.\n"

#: ../DistUpgrade/DistUpgradeController.py:324
#, python-format
msgid ""
"If you run a firewall, you may need to temporarily open this port. As this "
"is potentially dangerous it's not done automatically. You can open the port "
"with e.g.:\n"
"'%s'"
msgstr ""
"Viss du sit bak ein brannmur, må du kanskje opna denne porten mellombels. "
"Sia dette er potensielt farleg, vert det ikkje gjort automatisk. Du kan opna "
"porten med t.d. \n"
"'%s'"

#: ../DistUpgrade/DistUpgradeController.py:402
#: ../DistUpgrade/DistUpgradeController.py:426
msgid "Can not upgrade"
msgstr "Klarte ikkje å oppgradera"

#: ../DistUpgrade/DistUpgradeController.py:403
#, python-format
msgid "An upgrade from '%s' to '%s' is not supported with this tool."
msgstr "Å oppgradera frå «%s» til «%s» er ikkje støtta av dette verktøyet."

#: ../DistUpgrade/DistUpgradeController.py:427
msgid ""
"Your python3 install is corrupted. Please fix the '/usr/bin/python3' symlink."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:456
#, python-format
msgid "Can not write to '%s'"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:457
#, python-format
msgid ""
"Its not possible to write to the system directory '%s' on your system. The "
"upgrade can not continue.\n"
"Please make sure that the system directory is writable."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:468
msgid "Include latest updates from the Internet?"
msgstr "Inkluder dei siste oppdateringane frå Internett?"

#: ../DistUpgrade/DistUpgradeController.py:469
msgid ""
"The upgrade system can use the internet to automatically download the latest "
"updates and install them during the upgrade.  If you have a network "
"connection this is highly recommended.\n"
"\n"
"The upgrade will take longer, but when it is complete, your system will be "
"fully up to date.  You can choose not to do this, but you should install the "
"latest updates soon after upgrading.\n"
"If you answer 'no' here, the network is not used at all."
msgstr ""
"Oppgraderingsystemet kan bruka internett til å installera dei nyaste "
"oppdateringane under oppgraderinga. Har du internettsamband er dette sterkt "
"tilrådd.\n"
"\n"
"Oppgraderinga vil ta lenger tid, men systemet vil vera heilt oppdatert når "
"oppgraderinga er ferdig. Du kan velja ikkje å gjera dette, men du bør "
"installera dei siste oppdateringane så fort som mogleg etter oppgraderinga.\n"
"Svarar du «nei» her, vert ikkje nettet brukt i det heile."

#: ../DistUpgrade/DistUpgradeController.py:647
#, python-format
msgid "Not for humans during development stage of release %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:748
#, python-format
msgid "disabled on upgrade to %s"
msgstr "deaktivert under oppgradering til %s"

#: ../DistUpgrade/DistUpgradeController.py:805
msgid "No valid sources.list entry found"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:806
#, python-format
msgid ""
"While scanning your repository information no entry about %s could be "
"found.\n"
"\n"
"An upgrade might not succeed.\n"
"\n"
"Do you want to continue anyway?"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:818
msgid "No valid mirror found"
msgstr "Fann ikkje noko gyldig spegl"

#: ../DistUpgrade/DistUpgradeController.py:819
#, python-format
msgid ""
"While scanning your repository information no mirror entry for the upgrade "
"was found. This can happen if you run an internal mirror or if the mirror "
"information is out of date.\n"
"\n"
"Do you want to rewrite your 'sources.list' file anyway? If you choose 'Yes' "
"here it will update all '%s' to '%s' entries.\n"
"If you select 'No' the upgrade will cancel."
msgstr ""

#. hm, still nothing useful ...
#: ../DistUpgrade/DistUpgradeController.py:839
msgid "Generate default sources?"
msgstr "Vil du oppretta standardkjelder?"

#: ../DistUpgrade/DistUpgradeController.py:840
#, python-format
msgid ""
"After scanning your 'sources.list' no valid entry for '%s' was found.\n"
"\n"
"Should default entries for '%s' be added? If you select 'No', the upgrade "
"will cancel."
msgstr ""
"Ingen gyldige oppføringar for «%s» vart funne i «sources.list».\n"
"\n"
"Skal standardoppføringar for «%s» leggjast til? Vel du «Nei», vert "
"oppgraderinga avbroten."

#: ../DistUpgrade/DistUpgradeController.py:875
#: ../DistUpgrade/DistUpgradeController.py:881
msgid "Repository information invalid"
msgstr "Ugyldig arkivinformasjon"

#: ../DistUpgrade/DistUpgradeController.py:876
msgid ""
"Upgrading the repository information resulted in a invalid file so a bug "
"reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:882
msgid ""
"Upgrading the repository information resulted in a invalid file. To report a "
"bug install apport and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:891
msgid "Third party sources disabled"
msgstr "Tredjepartskjelder er deaktiverte"

#: ../DistUpgrade/DistUpgradeController.py:892
msgid ""
"Some third party entries in your sources.list were disabled. You can re-"
"enable them after the upgrade with the 'software-properties' tool or your "
"package manager."
msgstr ""
"Somme tredjeparts pakkekjelder verkar ikkje lenger. Du kan halda fram med å "
"bruka dei etter oppdateringa gjennom «Eigenskapar for programvare» eller med "
"Synaptic."

#: ../DistUpgrade/DistUpgradeController.py:934
msgid "Package in inconsistent state"
msgid_plural "Packages in inconsistent state"
msgstr[0] "Pakke i dårleg stand"
msgstr[1] "Pakkar i dårleg stand"

#: ../DistUpgrade/DistUpgradeController.py:937
#, python-format
msgid ""
"The package '%s' is in an inconsistent state and needs to be reinstalled, "
"but no archive can be found for it. Please reinstall the package manually or "
"remove it from the system."
msgid_plural ""
"The packages '%s' are in an inconsistent state and need to be reinstalled, "
"but no archive can be found for them. Please reinstall the packages manually "
"or remove them from the system."
msgstr[0] ""
"Pakken «%s» er ufullstendig og må installerast på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Installer pakken på nytt manuelt, eller fjern "
"han frå systemet."
msgstr[1] ""
"Pakkane «%s» er ufullstendige og må installerast på nytt, men ingen av dei "
"nødvendige arkiva vart funne. Installer pakkane på nytt manuelt, eller fjern "
"dei frå systemet."

#: ../DistUpgrade/DistUpgradeController.py:1000
msgid "Error during update"
msgstr "Feil under oppdatering"

#: ../DistUpgrade/DistUpgradeController.py:1001
msgid ""
"A problem occurred during the update. This is usually some sort of network "
"problem, please check your network connection and retry."
msgstr ""
"Eit problem oppstod under oppdateringa. Dette er vanlegvis ei form for "
"nettverksproblem. Sjekk nettverkstilkoplinga di og prøv om att."

#: ../DistUpgrade/DistUpgradeController.py:1044
#: ../DistUpgrade/DistUpgradeController.py:1049
msgid "EFI System Partition (ESP) not usable"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1045
msgid ""
"Your EFI System Partition (ESP) is not mounted at /boot/efi. Please ensure "
"that it is properly configured and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1050
msgid ""
"The EFI System Partition (ESP) mounted at /boot/efi is not writable. Please "
"mount this partition read-write and try again."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1057
msgid "Not enough free disk space"
msgstr "Ikkje nok ledig diskplass"

#. TRANSLATORS: you can change the order of the sentence,
#. make sure to keep all {str_*} string untranslated.
#: ../DistUpgrade/DistUpgradeController.py:1060
#, python-brace-format
msgid ""
"The upgrade has aborted. The upgrade needs a total of {str_total} free space "
"on disk '{str_dir}'. Please free at least an additional {str_needed} of disk "
"space on '{str_dir}'. {str_remedy}"
msgstr ""

#. specific ways to resolve lack of free space
#: ../DistUpgrade/DistUpgradeController.py:1065
msgid ""
"Remove temporary packages of former installations using 'sudo apt clean'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1067
msgid ""
"You can remove old kernels using 'sudo apt autoremove' and you could also "
"set COMPRESS=xz in /etc/initramfs-tools/initramfs.conf to reduce the size of "
"your initramfs."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1072
msgid ""
"Empty your trash and remove temporary packages of former installations using "
"'sudo apt-get clean'."
msgstr ""
"Tøm papirkorga, og fjern mellombelse pakkar med kommandoen «sudo apt-get "
"clean»."

#: ../DistUpgrade/DistUpgradeController.py:1075
msgid "Reboot to clean up files in /tmp."
msgstr "Start på nytt for å tømma mappa /tmp."

#. calc the dist-upgrade and see if the removals are ok/expected
#. do the dist-upgrade
#: ../DistUpgrade/DistUpgradeController.py:1114
#: ../DistUpgrade/DistUpgradeController.py:1990
#: ../DistUpgrade/DistUpgradeController.py:2040
msgid "Calculating the changes"
msgstr "Reknar ut endringane"

#. ask the user
#: ../DistUpgrade/DistUpgradeController.py:1158
msgid "Do you want to start the upgrade?"
msgstr "Vil du starta oppdateringa?"

#: ../DistUpgrade/DistUpgradeController.py:1177
#, python-format
msgid ""
"Livepatch security updates are not available for Ubuntu %s. If you upgrade, "
"Livepatch will turn off."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1249
msgid "Upgrade canceled"
msgstr "Oppgradering avbroten"

#: ../DistUpgrade/DistUpgradeController.py:1250
msgid ""
"The upgrade will cancel now and the original system state will be restored. "
"You can resume the upgrade at a later time."
msgstr ""
"Oppgraderinga vil no bli avbroten, og det opphavlege systemet vil verta "
"atterreist. Du kan venda tilbake til oppgraderinga når som helst."

#: ../DistUpgrade/DistUpgradeController.py:1256
#: ../DistUpgrade/DistUpgradeController.py:1401
msgid "Could not download the upgrades"
msgstr "Klarte ikkje å lasta ned oppgraderingane"

#: ../DistUpgrade/DistUpgradeController.py:1257
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. All files downloaded so far have been kept."
msgstr ""
"Oppgraderinga vart avbroten. Kontroller internettilkoplinga eller "
"installasjonsmediet og prøv på nytt. Filene som er lasta ned så langt er "
"tekne vare på."

#. FIXME: strings are not good, but we are in string freeze
#. currently
#: ../DistUpgrade/DistUpgradeController.py:1350
#: ../DistUpgrade/DistUpgradeController.py:1389
#: ../DistUpgrade/DistUpgradeController.py:1505
msgid "Error during commit"
msgstr "Feil under innsending"

#. generate a new cache
#: ../DistUpgrade/DistUpgradeController.py:1352
#: ../DistUpgrade/DistUpgradeController.py:1391
#: ../DistUpgrade/DistUpgradeController.py:1544
msgid "Restoring original system state"
msgstr "Gjenopprettar systemet til sin opphavlege tilstand"

#: ../DistUpgrade/DistUpgradeController.py:1353
#: ../DistUpgrade/DistUpgradeController.py:1368
#: ../DistUpgrade/DistUpgradeController.py:1392
msgid "Could not install the upgrades"
msgstr "Klarte ikkje å installera oppgraderingane"

#. invoke the frontend now and show a error message
#: ../DistUpgrade/DistUpgradeController.py:1358
msgid ""
"The upgrade has aborted. Your system could be in an unusable state. A "
"recovery will run now (dpkg --configure -a)."
msgstr ""
"Oppgraderinga vart avbroten. Systemet ditt kan vera ubrukeleg. Ei "
"gjenoppretting vert no køyrd (dpkg --configure -a)."

#: ../DistUpgrade/DistUpgradeController.py:1363
#, python-format
msgid ""
"\n"
"\n"
"Please report this bug in a browser at http://bugs.launchpad.net/ubuntu/"
"+source/ubuntu-release-upgrader/+filebug and attach the files in /var/log/"
"dist-upgrade/ to the bug report.\n"
"%s"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1402
msgid ""
"The upgrade has aborted. Please check your Internet connection or "
"installation media and try again. "
msgstr ""
"Oppgraderinga vart avbroten. Kontroller internett-tilkoplinga eller "
"installasjonsmediet og prøv på nytt. "

#: ../DistUpgrade/DistUpgradeController.py:1430
msgid "Searching for obsolete software"
msgstr "Søkjer etter utdatert programvare"

#: ../DistUpgrade/DistUpgradeController.py:1493
msgid "Remove obsolete packages?"
msgstr "Ønskjer du å fjerna utdaterte pakkar?"

#: ../DistUpgrade/DistUpgradeController.py:1494
#: ../data/gtkbuilder/DistUpgrade.ui.h:8
msgid "_Keep"
msgstr "_Ta vare på"

#: ../DistUpgrade/DistUpgradeController.py:1494
msgid "_Remove"
msgstr "_Fjern"

#: ../DistUpgrade/DistUpgradeController.py:1506
msgid ""
"A problem occurred during the clean-up. Please see the below message for "
"more information. "
msgstr ""
"Eit problem oppstod under oppryddinga. Sjå meldinga under for meir "
"informasjon. "

#. FIXME: instead of error out, fetch and install it
#. here
#: ../DistUpgrade/DistUpgradeController.py:1582
msgid "Required depends is not installed"
msgstr "Nødvendige avhengnader er ikkje installerte"

#: ../DistUpgrade/DistUpgradeController.py:1583
#, python-format
msgid "The required dependency '%s' is not installed. "
msgstr "Den nødvendige avhengnaden \"%s\" er ikkje installert. "

#. sanity check (check for ubuntu-desktop, brokenCache etc)
#. then open the cache (again)
#: ../DistUpgrade/DistUpgradeController.py:1850
#: ../DistUpgrade/DistUpgradeController.py:1940
msgid "Checking package manager"
msgstr "Sjekkar pakkehandsamaren"

#: ../DistUpgrade/DistUpgradeController.py:1856
#: ../DistUpgrade/DistUpgradeController.py:1862
msgid "Preparing the upgrade failed"
msgstr "Klarte ikkje å førebu oppgraderinga"

#: ../DistUpgrade/DistUpgradeController.py:1857
msgid ""
"Preparing the system for the upgrade failed so a bug reporting process is "
"being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1863
msgid ""
"Preparing the system for the upgrade failed. To report a bug install apport "
"and then execute 'apport-bug ubuntu-release-upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1880
#: ../DistUpgrade/DistUpgradeController.py:1890
msgid "Getting upgrade prerequisites failed"
msgstr "Klarte ikkje å lasta ned oppgraderingskrava"

#: ../DistUpgrade/DistUpgradeController.py:1881
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"Additionally, a bug reporting process is being started."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1891
msgid ""
"The system was unable to get the prerequisites for the upgrade. The upgrade "
"will abort now and restore the original system state.\n"
"\n"
"To report a bug install apport and then execute 'apport-bug ubuntu-release-"
"upgrader'."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1924
msgid "Updating repository information"
msgstr "Oppdaterer informasjon om arkivet"

#: ../DistUpgrade/DistUpgradeController.py:1931
msgid "Failed to add the cdrom"
msgstr "Klarte ikkje å leggja til CD-en"

#: ../DistUpgrade/DistUpgradeController.py:1932
msgid "Sorry, adding the cdrom was not successful."
msgstr "Beklagar, å leggja til CD-en var mislukka."

#: ../DistUpgrade/DistUpgradeController.py:1967
msgid "Invalid package information"
msgstr "Ugyldig pakkeinformasjon"

#: ../DistUpgrade/DistUpgradeController.py:1968
#, python-format
msgid ""
"After updating your package information, the essential package '%s' could "
"not be located. This may be because you have no official mirrors listed in "
"your software sources, or because of excessive load on the mirror you are "
"using. See /etc/apt/sources.list for the current list of configured software "
"sources.\n"
"In the case of an overloaded mirror, you may want to try the upgrade again "
"later."
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:1997
#: ../DistUpgrade/DistUpgradeController.py:2096
msgid "Fetching"
msgstr "Hentar"

#: ../DistUpgrade/DistUpgradeController.py:2004
#: ../DistUpgrade/DistUpgradeController.py:2025
#: ../DistUpgrade/DistUpgradeController.py:2046
#: ../DistUpgrade/DistUpgradeController.py:2100
msgid "Upgrading"
msgstr "Oppgraderer"

#: ../DistUpgrade/DistUpgradeController.py:2006
msgid "Upgrade infeasible"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2007
msgid ""
"The upgrade could not be completed, there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2028
msgid "Upgrade incomplete"
msgstr ""

#: ../DistUpgrade/DistUpgradeController.py:2029
msgid ""
"The upgrade has partially completed but there were errors during the upgrade "
"process."
msgstr ""

#. don't abort here, because it would restore the sources.list
#: ../DistUpgrade/DistUpgradeController.py:2051
#: ../DistUpgrade/DistUpgradeController.py:2102
#: ../DistUpgrade/DistUpgradeController.py:2108
#: ../DistUpgrade/DistUpgradeController.py:2119
msgid "Upgrade complete"
msgstr "Oppgradering fullført"

#: ../DistUpgrade/DistUpgradeController.py:2052
#: ../DistUpgrade/DistUpgradeController.py:2103
#: ../DistUpgrade/DistUpgradeController.py:2109
msgid ""
"The upgrade has completed but there were errors during the upgrade process."
msgstr "Oppgraderinga er ferdig, men nokre feil dukka opp under prosessen."

#: ../DistUpgrade/DistUpgradeController.py:2071
msgid "System upgrade is complete."
msgstr "Systemoppgraderinga er fullført."

#: ../DistUpgrade/DistUpgradeController.py:2120
msgid "The partial upgrade was completed."
msgstr "Delvis oppgradering er ferdig."

#: ../DistUpgrade/DistUpgradeController.py:2148
msgid "Lock screen disabled"
msgstr "Skjermlåsing slått av"

#: ../DistUpgrade/DistUpgradeController.py:2149
msgid ""
"Your lock screen has been disabled and will remain disabled until you reboot."
msgstr "Skjermlåsing er slått av fram til du startar maskina på nytt."

#: ../DistUpgrade/DistUpgradeFetcher.py:107
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:170
msgid "Could not find the release notes"
msgstr "Klarte ikkje å finna versjonsnotata"

#: ../DistUpgrade/DistUpgradeFetcher.py:108
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:171
msgid "The server may be overloaded. "
msgstr "Tenaren kan vera overbelasta. "

#: ../DistUpgrade/DistUpgradeFetcher.py:120
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:175
msgid "Could not download the release notes"
msgstr "Klarte ikkje å lasta ned versjonsnotata"

#: ../DistUpgrade/DistUpgradeFetcher.py:121
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:176
msgid "Please check your internet connection."
msgstr "Sjekk internettilkoplinga di."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:73
#, python-format
msgid "authenticate '%(file)s' against '%(signature)s' "
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:97
#, python-format
msgid "extracting '%s'"
msgstr "pakkar ut «%s»"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:118
#: ../DistUpgrade/DistUpgradeFetcherCore.py:119
msgid "Could not run the upgrade tool"
msgstr "Klarte ikkje å køyra oppgraderingsverktøyet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:120
msgid ""
"This is most likely a bug in the upgrade tool. Please report it as a bug "
"using the command 'ubuntu-bug ubuntu-release-upgrader-core'."
msgstr ""

#: ../DistUpgrade/DistUpgradeFetcherCore.py:197
msgid "Upgrade tool signature"
msgstr "Signaturen til oppgraderingsvertøyet"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:204
msgid "Upgrade tool"
msgstr "Oppgraderingsvertøy"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:238
msgid "Failed to fetch"
msgstr "Klarte ikkje å henta"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:239
msgid "Fetching the upgrade failed. There may be a network problem. "
msgstr ""
"Klarte ikkje å henta oppgraderinga. Det kan vera eit nettverksproblem. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:243
msgid "Authentication failed"
msgstr "Å stadfesta identiteten mislukkast"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:244
msgid ""
"Authenticating the upgrade failed. There may be a problem with the network "
"or with the server. "
msgstr ""
"Klarte ikkje å stadfesta identiteten til oppgraderinga. Det kan vere feil "
"med nettverket eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:249
msgid "Failed to extract"
msgstr "Mislukkast i å pakka ut"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:250
msgid ""
"Extracting the upgrade failed. There may be a problem with the network or "
"with the server. "
msgstr ""
"Klarte ikkje å pakka ut oppgraderinga. Det kan vera eit problem med "
"netverket eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:255
msgid "Verification failed"
msgstr "Verifiseringsfeil"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:256
msgid ""
"Verifying the upgrade failed.  There may be a problem with the network or "
"with the server. "
msgstr ""
"Å verifisera oppgraderinga mislukkast. Det kan vera ein feil med nettverket "
"eller tenaren. "

#: ../DistUpgrade/DistUpgradeFetcherCore.py:270
#: ../DistUpgrade/DistUpgradeFetcherCore.py:276
msgid "Can not run the upgrade"
msgstr "Klarte ikkje å køyra oppgradering"

#: ../DistUpgrade/DistUpgradeFetcherCore.py:271
msgid ""
"This usually is caused by a system where /tmp is mounted noexec. Please "
"remount without noexec and run the upgrade again."
msgstr ""
"Dette skjer vanlegvis i system der mappa /tmp er montert med køyreløyve. "
"Monter på nytt utan køyreløyve og køyr oppgraderinga på nytt."

#: ../DistUpgrade/DistUpgradeFetcherCore.py:277
#, python-format
msgid "The error message is '%s'."
msgstr "Feilmeldinga er «%s»."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:151
msgid "&Upgrade"
msgstr "&Oppgrader"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:154
#: ../DistUpgrade/DistUpgradeViewKDE.py:715
msgid "&Cancel"
msgstr "&Avbryt"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:156
#: ../data/gtkbuilder/ReleaseNotes.ui.h:1
msgid "Release Notes"
msgstr "Versjonsnotat"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:191
msgid "Upgrade"
msgstr "Oppgrader"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:207
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:222
msgid "Downloading additional package files..."
msgstr "Lastar ned ekstra pakkefiler..."

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:224
#, python-format
msgid "File %s of %s at %sB/s"
msgstr "Fil %s av %s, %sB/s"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:228
#, python-format
msgid "File %s of %s"
msgstr "Fil %s av %s"

#. print("mediaChange %s %s" % (medium, drive))
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:235
#: ../DistUpgrade/DistUpgradeViewGtk3.py:125
#: ../DistUpgrade/DistUpgradeViewKDE.py:302
#, python-format
msgid "Please insert '%s' into the drive '%s'"
msgstr "Sett '%s' i stasjonen '%s'"

#: ../DistUpgrade/DistUpgradeFetcherKDE.py:237
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:242
#: ../DistUpgrade/DistUpgradeFetcherKDE.py:243
#: ../DistUpgrade/DistUpgradeViewKDE.py:303
msgid "Media Change"
msgstr "Medieendring"

#: ../DistUpgrade/DistUpgradeQuirks.py:219
msgid "Your graphics hardware may not be fully supported in Ubuntu 14.04."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:221
msgid ""
"Running the 'unity' desktop environment is not fully supported by your "
"graphics hardware. You will maybe end up in a very slow environment after "
"the upgrade. Our advice is to keep the LTS version for now. For more "
"information see https://wiki.ubuntu.com/X/Bugs/"
"UpdateManagerWarningForUnity3D Do you still want to continue with the "
"upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:245
msgid "Your graphics hardware may not be fully supported in Ubuntu 12.04 LTS."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:247
msgid ""
"The support in Ubuntu 12.04 LTS for your Intel graphics hardware is limited "
"and you may encounter problems after the upgrade. For more information see "
"https://wiki.ubuntu.com/X/Bugs/UpdateManagerWarningForI8xx Do you want to "
"continue with the upgrade?"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:267
msgid ""
"Upgrading may reduce desktop effects, and performance in games and other "
"graphically intensive programs."
msgstr ""
"Å oppgradera kan redusera skrivebordseffekter, samt ytinga i spel og andre "
"grafikkintensive program."

#: ../DistUpgrade/DistUpgradeQuirks.py:271
msgid ""
"This computer is currently using the AMD 'fglrx' graphics driver. No version "
"of this driver is available that works with your hardware in Ubuntu 10.04 "
"LTS.\n"
"\n"
"Do you want to continue?"
msgstr ""
"Datamaskina brukar AMD «fglrx»-grafikkdrivaren. Ingen tilgjengelege "
"versjonar av denne drivaren verkar med maskinvaren din i Ubuntu 10.04 LTS."

#: ../DistUpgrade/DistUpgradeQuirks.py:310
msgid "Sorry, no more upgrades for this system"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:311
#, python-format
msgid ""
"There will not be any further Ubuntu releases for this system's 'i386' "
"architecture.\n"
"\n"
"Updates for Ubuntu %s will continue until %s."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:326
msgid "No ARMv6 CPU"
msgstr "Ingen ARMv6-prosessor"

#: ../DistUpgrade/DistUpgradeQuirks.py:327
msgid ""
"Your system uses an ARM CPU that is older than the ARMv6 architecture. All "
"packages in karmic were built with optimizations requiring ARMv6 as the "
"minimal architecture. It is not possible to upgrade your system to a new "
"Ubuntu release with this hardware."
msgstr ""
"Systemet brukar ein ARM-prosessor som er eldre enn ARMv6-arkitekturen. Alle "
"pakkane i karmic krev minimum ARMv6. Det er ikkje mogleg å oppgradera "
"systemet ditt til den nyaste Ubuntu-utgjevinga med den noverande maskinvaren."

#: ../DistUpgrade/DistUpgradeQuirks.py:347
msgid "No init available"
msgstr "Ingen oppstartsprogram (init) er tilgjengelege"

#: ../DistUpgrade/DistUpgradeQuirks.py:348
msgid ""
"Your system appears to be a virtualised environment without an init daemon, "
"e.g. Linux-VServer. Ubuntu 10.04 LTS cannot function within this type of "
"environment, requiring an update to your virtual machine configuration "
"first.\n"
"\n"
"Are you sure you want to continue?"
msgstr ""
"Systemet ditt ser ut til å vera i eit virtualisert miljø utan "
"oppstartsprogram (init), til dømes Linux-VServer. Ubuntu 10.04 LTS fungerer "
"ikkje i slike miljø, og krev ei oppdatering av innstillingane i den "
"virtuelle maskina først.\n"
"\n"
"Er du sikker på at du vil halda fram?"

#: ../DistUpgrade/DistUpgradeQuirks.py:461
#, fuzzy
msgid "Connection to the Snap Store failed"
msgstr "Klarte ikkje kopla til Snap Store"

#: ../DistUpgrade/DistUpgradeQuirks.py:462
msgid ""
"You have the package lxd installed but your system is unable to reach the "
"Snap Store. lxd is now provided via a snap and the release upgrade will fail "
"if snapd is not functional. Please make sure you're connected to the "
"Internet and update any firewall or proxy settings as needed so that you can "
"reach api.snapcraft.io. If you are an enterprise with a firewall setup you "
"may want to configure a Snap Store proxy."
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:477
msgid "Connection to Snap Store failed"
msgstr "Klarte ikkje kopla til Snap Store"

#: ../DistUpgrade/DistUpgradeQuirks.py:478
msgid ""
"Your system does not have a connection to the Snap Store. For the best "
"upgrade experience make sure that your system can connect to api.snapcraft."
"io.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Systemet klarte ikkje kopla til Snap-programvaresenteret. For ei best muleg "
"oppgradering bør du stadfesta at systemet kan kopla til api.snapcraft.io.\n"
"Vil du likevel halda fram med oppgraderinga?"

#: ../DistUpgrade/DistUpgradeQuirks.py:487
msgid "Outdated snapd package"
msgstr "Utdatert snapd-pakke"

#: ../DistUpgrade/DistUpgradeQuirks.py:488
msgid ""
"Your system does not have the latest version of snapd. Please update the "
"version of snapd on your system to improve the upgrade experience.\n"
"Do you still want to continue with the upgrade?"
msgstr ""
"Systemet ditt har ikkje den nyaste versjonen av snapd. For ei best muleg "
"oppgradering bør du oppdatera snapd-pakken.\n"
"Vil du likevel halda fram med oppgraderinga?"

#. now perform direct API calls to the store, requesting size
#. information for each of the snaps needing installation
#: ../DistUpgrade/DistUpgradeQuirks.py:511
msgid "Calculating snap size requirements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:542
msgid "Processing snap replacements"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:548
#, python-format
msgid "refreshing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:552
#, python-format
msgid "removing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:555
#, python-format
msgid "installing snap %s"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:584
msgid "PAE not enabled"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:585
msgid ""
"Your system uses a CPU that does not have PAE enabled. Ubuntu only supports "
"non-PAE systems up to Ubuntu 12.04. To upgrade to a later version of Ubuntu, "
"you must enable PAE (if this is possible) see:\n"
"http://help.ubuntu.com/community/EnablingPAE"
msgstr ""

#: ../DistUpgrade/DistUpgradeQuirks.py:895
msgid "Checking for installed snaps"
msgstr "Sjekkar installerte snap-pakkar"

#: ../DistUpgrade/DistUpgradeMain.py:58
msgid "Use the given path to search for a cdrom with upgradable packages"
msgstr ""
"Bruk den oppgjevne stien til å søkja etter ei CD-plate med pakkar som kan "
"oppgraderast"

#: ../DistUpgrade/DistUpgradeMain.py:64
msgid ""
"Use frontend. Currently available: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"
msgstr ""
"Bruk eit grafisk grensesnitt. Tilgjengelege er: \n"
"DistUpgradeViewText, DistUpgradeViewGtk, DistUpgradeViewKDE"

#: ../DistUpgrade/DistUpgradeMain.py:67
msgid "*DEPRECATED* this option will be ignored"
msgstr "*FORELDA* dette valet vil verta ignorert"

#: ../DistUpgrade/DistUpgradeMain.py:70
msgid "Perform a partial upgrade only (no sources.list rewriting)"
msgstr ""
"Gjennomfør berre ei delevis oppgradering (inga omskriving av sources.list)"

#: ../DistUpgrade/DistUpgradeMain.py:73
msgid "Disable GNU screen support"
msgstr "Skru av GNU-skjermstøtte"

#: ../DistUpgrade/DistUpgradeMain.py:75
msgid "Set datadir"
msgstr "Vel datamappe"

#: ../DistUpgrade/DistUpgradeMain.py:78
msgid "Upgrade to the development release"
msgstr "Oppgrader til utviklingsversjonen"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:146
#: ../DistUpgrade/DistUpgradeViewKDE.py:316
msgid "Fetching is complete"
msgstr "Ferdig med å henta"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:161
#: ../DistUpgrade/DistUpgradeViewKDE.py:332
#, python-format
msgid "Fetching file %li of %li at %sB/s"
msgstr "Hentar fil %li av %li ved %sB/s"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:163
#: ../DistUpgrade/DistUpgradeViewGtk3.py:325
#: ../DistUpgrade/DistUpgradeViewKDE.py:333
#: ../DistUpgrade/DistUpgradeViewKDE.py:483
#, python-format
msgid "About %s remaining"
msgstr "Omtrent %s står att"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:166
#: ../DistUpgrade/DistUpgradeViewKDE.py:335
#, python-format
msgid "Fetching file %li of %li"
msgstr "Hentar fil %li av %li"

#. FIXME: add support for the timeout
#. of the terminal (to display something useful then)
#. -> longer term, move this code into python-apt
#: ../DistUpgrade/DistUpgradeViewGtk3.py:198
#: ../DistUpgrade/DistUpgradeViewKDE.py:373
msgid "Applying changes"
msgstr "Utfører endringar"

#. we do not report followup errors from earlier failures
#: ../DistUpgrade/DistUpgradeViewGtk3.py:224
#: ../DistUpgrade/DistUpgradeViewKDE.py:386
msgid "dependency problems - leaving unconfigured"
msgstr "problem med avhengnader – set ikkje opp pakken"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:229
#: ../DistUpgrade/DistUpgradeViewKDE.py:388
#, python-format
msgid "Could not install '%s'"
msgstr "Kunne ikkje installere '%s'"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:230
#: ../DistUpgrade/DistUpgradeViewKDE.py:389
#, python-format
msgid ""
"The upgrade will continue but the '%s' package may not be in a working "
"state. Please consider submitting a bug report about it."
msgstr ""
"Oppgraderinga vil halda fram, men pakken «%s» vil kanskje ikkje fungera. "
"Vurder å senda inn ei feilmelding om dette."

#. self.expander.set_expanded(True)
#: ../DistUpgrade/DistUpgradeViewGtk3.py:247
#: ../DistUpgrade/DistUpgradeViewKDE.py:411
#, python-format
msgid ""
"Replace the customized configuration file\n"
"'%s'?"
msgstr ""
"Vil du erstatta den eigendefinerte oppsettsfila\n"
"\"%s\"?"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:248
#: ../DistUpgrade/DistUpgradeViewKDE.py:412
msgid ""
"You will lose any changes you have made to this configuration file if you "
"choose to replace it with a newer version."
msgstr ""
"Viss du byter ut oppsettsfila med ein nyare versjon, vil du mista endringane "
"du har gjort i fila."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:268
#: ../DistUpgrade/DistUpgradeViewKDE.py:435
msgid "The 'diff' command was not found"
msgstr "Kommandoen \"diff\" vart ikkje funnen"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:535
#: ../DistUpgrade/DistUpgradeViewText.py:140
msgid "A fatal error occurred"
msgstr "Ein kritisk feil oppstod"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:536
msgid ""
"Please report this as a bug (if you haven't already) and include the files /"
"var/log/dist-upgrade/main.log and /var/log/dist-upgrade/apt.log in your "
"report. The upgrade has aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter dette som ein feil (om du ikkje allereie har gjort det), og "
"inkluder filene /var/log/dist-upgrade/main.log og /var/log/dist-upgrade/apt."
"log i rapporten. Oppgraderinga er avbroten.\n"
"Den opphavlege sources.list er lagra i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:566
msgid "Ctrl-c pressed"
msgstr "Ctrl + C trykt"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:567
msgid ""
"This will abort the operation and may leave the system in a broken state. "
"Are you sure you want to do that?"
msgstr ""
"Dette vil avbryta operasjonen og kanskje setja systemet ditt i ein ubrukeleg "
"tilstand. Er du sikker på at du vil gjera dette?"

#. append warning
#: ../DistUpgrade/DistUpgradeViewGtk3.py:706
msgid "To prevent data loss close all open applications and documents."
msgstr "For å hindra tap av data, bør du lukke alle program og dokument."

#: ../DistUpgrade/DistUpgradeViewGtk3.py:720
#, python-format
msgid "No longer supported by Canonical (%s)"
msgstr "Ikkje lenger støtta av Canonical (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:721
#, python-format
msgid "<b>Downgrade (%s)</b>"
msgstr "<b>Nedgrader (%s)</b>"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:722
#, python-format
msgid "Remove (%s)"
msgstr "Fjern (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:723
#, python-format
msgid "No longer needed (%s)"
msgstr "Ikkje lenger nødvendig (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:724
#, python-format
msgid "Install (%s)"
msgstr "Installer (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:725
#, python-format
msgid "Upgrade (%s)"
msgstr "Oppgrader (%s)"

#: ../DistUpgrade/DistUpgradeViewGtk3.py:773
msgid "Cancel"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewGtk3.py:774
#: ../DistUpgrade/DistUpgradeViewKDE.py:939
msgid "Continue"
msgstr ""

#: ../DistUpgrade/DistUpgradeViewKDE.py:447
msgid "Show Difference >>>"
msgstr "Vis forskjellar >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:450
msgid "<<< Hide Difference"
msgstr "<<< Skjul forskjellar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:701
msgid "Error"
msgstr "Feil"

#: ../DistUpgrade/DistUpgradeViewKDE.py:719
#: ../DistUpgrade/DistUpgradeViewKDE.py:960
msgid "&Close"
msgstr "&Lukk"

#: ../DistUpgrade/DistUpgradeViewKDE.py:753
msgid "Show Terminal >>>"
msgstr "Syn terminal >>>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:756
msgid "<<< Hide Terminal"
msgstr "<<< Gøym terminal"

#: ../DistUpgrade/DistUpgradeViewKDE.py:829
msgid "Information"
msgstr "Informasjon"

#: ../DistUpgrade/DistUpgradeViewKDE.py:877
msgid "&Start Upgrade"
msgstr "&Start oppgradering"

#: ../DistUpgrade/DistUpgradeViewKDE.py:879
#: ../DistUpgrade/DistUpgradeViewKDE.py:923
#: ../DistUpgrade/DistUpgradeViewKDE.py:926
#: ../data/gtkbuilder/DistUpgrade.ui.h:7
msgid "Details"
msgstr "Detaljar"

#: ../DistUpgrade/DistUpgradeViewKDE.py:901
#, python-format
msgid "No longer supported %s"
msgstr "Ikkje lenger støtta %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:903
#, python-format
msgid "Remove %s"
msgstr "Fjern %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:905
#: ../DistUpgrade/DistUpgradeViewText.py:239
#, python-format
msgid "Remove (was auto installed) %s"
msgstr "Fjern (vart installert automatisk) %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:907
#, python-format
msgid "Install %s"
msgstr "Installer %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:909
#, python-format
msgid "Upgrade %s"
msgstr "Oppgrader %s"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
#: ../DistUpgrade/DistUpgradeViewText.py:292
msgid "Restart required"
msgstr "Omstart er nødvendig"

#: ../DistUpgrade/DistUpgradeViewKDE.py:956
msgid "<b><big>Restart the system to complete the upgrade</big></b>"
msgstr "<b><big>Start systemet på ny for å fullføra oppgraderinga</big></b>"

#: ../DistUpgrade/DistUpgradeViewKDE.py:959
#: ../data/gtkbuilder/DistUpgrade.ui.h:14
msgid "_Restart Now"
msgstr "_Start på nytt no"

#. FIXME make this user friendly
#: ../DistUpgrade/DistUpgradeViewKDE.py:977
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly advised to resume the upgrade."
msgstr ""
"<b><big>Vil du avbryte den køyrande oppgraderinga?</big></big>\n"
"\n"
"Systemet kan verta ubrukeleg viss du avbryt oppgraderinga. Det er sterkt "
"tilrådd å fullføra oppgraderinga."

#: ../DistUpgrade/DistUpgradeViewKDE.py:981
msgid "Cancel Upgrade?"
msgstr "Avbryt oppgradering?"

#: ../DistUpgrade/DistUpgradeView.py:82
#, python-format
msgid "%li day"
msgid_plural "%li days"
msgstr[0] "%li dag"
msgstr[1] "%li dagar"

#: ../DistUpgrade/DistUpgradeView.py:84
#, python-format
msgid "%li hour"
msgid_plural "%li hours"
msgstr[0] "%li time"
msgstr[1] "%li timar"

#: ../DistUpgrade/DistUpgradeView.py:86
#, python-format
msgid "%li minute"
msgid_plural "%li minutes"
msgstr[0] "%li minutt"
msgstr[1] "%li minutt"

#: ../DistUpgrade/DistUpgradeView.py:87
#, python-format
msgid "%li second"
msgid_plural "%li seconds"
msgstr[0] "%li sekund"
msgstr[1] "%li sekund"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_days)s %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:103
#, python-format
msgid "%(str_days)s %(str_hours)s"
msgstr "%(str_days)s %(str_hours)s"

#. TRANSLATORS: you can alter the ordering of the remaining time
#. information here if you shuffle %(str_hours)s %(str_minutes)s
#. around. Make sure to keep all '$(str_*)s' in the translated string
#. and do NOT change anything appart from the ordering.
#.
#. %(str_hours)s will be either "1 hour" or "2 hours" depending on the
#. plural form
#.
#. Note: most western languages will not need to change this
#: ../DistUpgrade/DistUpgradeView.py:121
#, python-format
msgid "%(str_hours)s %(str_minutes)s"
msgstr "%(str_hours)s %(str_minutes)s"

#. 56 kbit
#. 1Mbit = 1024 kbit
#: ../DistUpgrade/DistUpgradeView.py:172
#, python-format
msgid ""
"This download will take about %s with a 1Mbit DSL connection and about %s "
"with a 56k modem."
msgstr ""
"Nedlastinga vil ta omlag %s med ei DSL-tilkopling på 1 Mbit og omlag %s med "
"eit 56k-modem."

#. if we have a estimated speed, use it
#: ../DistUpgrade/DistUpgradeView.py:176
#, python-format
msgid "This download will take about %s with your connection. "
msgstr "Denne nedlastinga vil ta ca. %s med tilkoplinga di. "

#. Declare these translatable strings from the .ui files here so that
#. xgettext picks them up.
#: ../DistUpgrade/DistUpgradeView.py:270 ../data/gtkbuilder/DistUpgrade.ui.h:22
msgid "Preparing to upgrade"
msgstr "Førebur oppgraderinga"

#: ../DistUpgrade/DistUpgradeView.py:271
msgid "Getting new software channels"
msgstr "Hentar nye programvarekanalar"

#: ../DistUpgrade/DistUpgradeView.py:272 ../data/gtkbuilder/DistUpgrade.ui.h:24
msgid "Getting new packages"
msgstr "Hentar nye programvarepakkar"

#: ../DistUpgrade/DistUpgradeView.py:273 ../data/gtkbuilder/DistUpgrade.ui.h:27
msgid "Installing the upgrades"
msgstr "Installerer oppgraderingane"

#: ../DistUpgrade/DistUpgradeView.py:274 ../data/gtkbuilder/DistUpgrade.ui.h:26
msgid "Cleaning up"
msgstr "Ryddar opp"

#: ../DistUpgrade/DistUpgradeView.py:363
#, python-format
msgid ""
"%(amount)d installed package is no longer supported by Canonical. You can "
"still get support from the community."
msgid_plural ""
"%(amount)d installed packages are no longer supported by Canonical. You can "
"still get support from the community."
msgstr[0] ""
"%(amount)d installert pakke er ikkje lenger støtta av Canonical. Du kan "
"framleis få støtte frå fellesskapet."
msgstr[1] ""
"%(amount)d installerte pakkar er ikkje lenger støtta av Canonical. Du kan "
"framleis få støtte frå fellesskapet."

#. FIXME: make those two separate lines to make it clear
#. that the "%" applies to the result of ngettext
#: ../DistUpgrade/DistUpgradeView.py:372
#, python-format
msgid "%d package is going to be removed."
msgid_plural "%d packages are going to be removed."
msgstr[0] "%d-pakka vil fjernast."
msgstr[1] "%d pakkar vil fjernast."

#: ../DistUpgrade/DistUpgradeView.py:377
#, python-format
msgid "%d new package is going to be installed."
msgid_plural "%d new packages are going to be installed."
msgstr[0] "%d ny pakke vert installert."
msgstr[1] "%d nye pakkar vert installerte."

#: ../DistUpgrade/DistUpgradeView.py:383
#, python-format
msgid "%d package is going to be upgraded."
msgid_plural "%d packages are going to be upgraded."
msgstr[0] "%d-pakka vil verta oppgradert."
msgstr[1] "%d pakkar vil verta oppgraderte."

#: ../DistUpgrade/DistUpgradeView.py:391
#, python-format
msgid ""
"\n"
"\n"
"You have to download a total of %s. "
msgstr ""
"\n"
"\n"
"Du må laste ned totalt %s. "

#: ../DistUpgrade/DistUpgradeView.py:396
msgid ""
"Installing the upgrade can take several hours. Once the download has "
"finished, the process cannot be canceled."
msgstr ""
"Oppgraderinga kan ta fleire timar. Du kan ikkje avbryta oppgraderinga etter "
"at nedlastingane er fullførte."

#: ../DistUpgrade/DistUpgradeView.py:400
msgid ""
"Fetching and installing the upgrade can take several hours. Once the "
"download has finished, the process cannot be canceled."
msgstr ""
"Nedlastinga og oppgraderinga kan ta fleire timar. Du kan ikkje avbryta "
"oppgraderinga etter at nedlastingane er fullførte."

#: ../DistUpgrade/DistUpgradeView.py:405
msgid "Removing the packages can take several hours. "
msgstr "Det kan ta fleire timar å fjerna pakkane. "

#. FIXME: this should go into DistUpgradeController
#: ../DistUpgrade/DistUpgradeView.py:410
msgid "The software on this computer is up to date."
msgstr "Programvaren på maskina er oppdatert."

#: ../DistUpgrade/DistUpgradeView.py:411
msgid ""
"There are no upgrades available for your system. The upgrade will now be "
"canceled."
msgstr ""
"Det er ingen oppgraderingar tilgjengelege for ditt system. Oppgraderinga vil "
"no verta avbroten."

#: ../DistUpgrade/DistUpgradeView.py:427
msgid "Reboot required"
msgstr "Omstart er nødvendig"

#: ../DistUpgrade/DistUpgradeView.py:428
msgid ""
"The upgrade is finished and a reboot is required. Do you want to do this now?"
msgstr ""
"Oppgraderinga er ikkje fullført og ein omstart er nødvendig. Vil du gjera "
"dette no?"

#: ../DistUpgrade/DistUpgradeViewText.py:82
#, python-format
msgid "Progress: [%3i%%]"
msgstr "Framdrift: [%3i%%]"

#: ../DistUpgrade/DistUpgradeViewText.py:141
msgid ""
"Please report this as a bug and include the files /var/log/dist-upgrade/main."
"log and /var/log/dist-upgrade/apt.log in your report. The upgrade has "
"aborted.\n"
"Your original sources.list was saved in /etc/apt/sources.list.distUpgrade."
msgstr ""
"Rapporter dette som ein feil og inkluder filene /var/log/dist-upgrade/main."
"log og /var/log/dist-upgrade/apt.log i rapporten. Oppgraderinga er "
"avbroten. \n"
"Den opphavlege sources.list er lagra i /etc/apt/sources.list.distUpgrade."

#: ../DistUpgrade/DistUpgradeViewText.py:165
msgid "Aborting"
msgstr "Avbryt"

#: ../DistUpgrade/DistUpgradeViewText.py:171
msgid "Demoted:\n"
msgstr "Degradert:\n"

#: ../DistUpgrade/DistUpgradeViewText.py:178
msgid "To continue please press [ENTER]"
msgstr "Trykk Enter for å halda fram"

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
#: ../DistUpgrade/DistUpgradeViewText.py:261
msgid "Continue [yN] "
msgstr "Hald fram [jN] "

#: ../DistUpgrade/DistUpgradeViewText.py:215
#: ../DistUpgrade/DistUpgradeViewText.py:253
msgid "Details [d]"
msgstr "Detaljar [d]"

#. TRANSLATORS: the "y" is "yes"
#. TRANSLATORS: first letter of a positive (yes) answer
#: ../DistUpgrade/DistUpgradeViewText.py:219
#: ../DistUpgrade/DistUpgradeViewText.py:264
msgid "y"
msgstr "j"

#. TRANSLATORS: the "n" is "no"
#. TRANSLATORS: first letter of a negative (no) answer
#: ../DistUpgrade/DistUpgradeViewText.py:222
#: ../DistUpgrade/DistUpgradeViewText.py:271
msgid "n"
msgstr "n"

#. TRANSLATORS: the "d" is "details"
#: ../DistUpgrade/DistUpgradeViewText.py:225
msgid "d"
msgstr "d"

#: ../DistUpgrade/DistUpgradeViewText.py:230
#, python-format
msgid "No longer supported: %s\n"
msgstr "Ikkje lenger støtta: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:235
#, python-format
msgid "Remove: %s\n"
msgstr "Fjern %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:245
#, python-format
msgid "Install: %s\n"
msgstr "Installer: %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:250
#, python-format
msgid "Upgrade: %s\n"
msgstr "Oppgrader %s\n"

#: ../DistUpgrade/DistUpgradeViewText.py:268
msgid "Continue [Yn] "
msgstr "Hald fram [Jn] "

#: ../DistUpgrade/DistUpgradeViewText.py:293
msgid ""
"To finish the upgrade, a restart is required.\n"
"If you select 'y' the system will be restarted."
msgstr ""
"For å fullføra oppgraderinga, er ein omstart av datamaskina nødvendig.\n"
"Dersom du svarar 'j', vil datamaskinen starta på nytt."

#: ../DistUpgrade/GtkProgress.py:71
#, python-format
msgid "Downloading file %(current)li of %(total)li with %(speed)s/s"
msgstr "Lastar ned fil %(current)li av %(total)li med %(speed)s/s"

#: ../DistUpgrade/GtkProgress.py:77
#, python-format
msgid "Downloading file %(current)li of %(total)li"
msgstr "Lastar ned fila %(current)li av %(total)li"

#: ../data/gtkbuilder/AcquireProgress.ui.h:1
#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:7
msgid "Show progress of individual files"
msgstr "Syn framdrift for enkeltfiler"

#: ../data/gtkbuilder/DistUpgrade.ui.h:1
msgid "_Cancel Upgrade"
msgstr "_Avbryt oppgradering"

#: ../data/gtkbuilder/DistUpgrade.ui.h:2
msgid "_Resume Upgrade"
msgstr "_Forsett oppgraderinga"

#: ../data/gtkbuilder/DistUpgrade.ui.h:3
msgid ""
"<b><big>Cancel the running upgrade?</big></b>\n"
"\n"
"The system could be in an unusable state if you cancel the upgrade. You are "
"strongly adviced to resume the upgrade."
msgstr ""
"<b><big>Vil du avbryta oppgraderinga?</big></b>\n"
"Systemet kan bli ubrukeleg om du avbryt oppgraderinga. Du er oppmoda på det "
"sterkaste å forsetja oppgraderinga."

#: ../data/gtkbuilder/DistUpgrade.ui.h:6
msgid "_Start Upgrade"
msgstr "_Start oppgradering"

#: ../data/gtkbuilder/DistUpgrade.ui.h:9
msgid "_Replace"
msgstr "_Erstatt"

#: ../data/gtkbuilder/DistUpgrade.ui.h:10
msgid "Difference between the files"
msgstr "Forskjellar mellom filene"

#: ../data/gtkbuilder/DistUpgrade.ui.h:11
msgid "_Report Bug"
msgstr "_Rapporter ein feil"

#: ../data/gtkbuilder/DistUpgrade.ui.h:12
msgid "_Continue"
msgstr "_Hald fram"

#: ../data/gtkbuilder/DistUpgrade.ui.h:13
msgid "<b><big>Start the upgrade?</big></b>"
msgstr "<b><big>Start oppgraderinga?</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:15
msgid ""
"<b><big>Restart the system to complete the upgrade</big></b>\n"
"\n"
"Please save your work before continuing."
msgstr "<b><big>Start systemet på nytt for å fullføra oppgraderinga</big></b>"

#: ../data/gtkbuilder/DistUpgrade.ui.h:18
msgid "Distribution Upgrade"
msgstr "Distribusjonsoppgradering"

#: ../data/gtkbuilder/DistUpgrade.ui.h:20
#, no-c-format
msgid "<b><big>Upgrading Ubuntu to version %s</big></b>"
msgstr ""

#: ../data/gtkbuilder/DistUpgrade.ui.h:21
msgid "    "
msgstr "    "

#: ../data/gtkbuilder/DistUpgrade.ui.h:23
msgid "Setting new software channels"
msgstr "Endrar kanalene for programvare"

#: ../data/gtkbuilder/DistUpgrade.ui.h:25
msgid "Restarting the computer"
msgstr "Startar systemet på nytt"

#: ../data/gtkbuilder/DistUpgrade.ui.h:28
msgid "Terminal"
msgstr "Terminal"

#: ../data/gtkbuilder/ReleaseNotes.ui.h:2
msgid "_Upgrade"
msgstr "_Oppgrader"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:1
msgid "<b>A new version of Ubuntu is available. Would you like to upgrade?</b>"
msgstr "<b>Ein ny versjon av Ubuntu er tilgjengeleg. Vil du oppgradera?</b>"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:2
msgid "Don't Upgrade"
msgstr "Ikkje oppgrader"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:3
msgid "Ask Me Later"
msgstr "Spør meg seinare"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:4
msgid "Yes, Upgrade Now"
msgstr "Oppgrader no"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:5
msgid "You have declined to upgrade to the new Ubuntu"
msgstr "Du har takka nei til å oppgradera til eit nytt Ubuntu"

#: ../data/gtkbuilder/UpgradePromptDialog.ui.h:6
msgid ""
"You can upgrade at a later time by opening Software Updater and click on "
"\"Upgrade\"."
msgstr ""
"Du kan oppgradera seinare ved å opna oppdateringsverktøyet og velja "
"«Oppgrader»."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:1
msgid "Perform a release upgrade"
msgstr "Utfør oppgradering av utgåva"

#: ../data/com.ubuntu.release-upgrader.policy.in.h:2
msgid "To upgrade Ubuntu, you need to authenticate."
msgstr "Du må autentisera deg for å oppgradera Ubuntu."

#: ../data/com.ubuntu.release-upgrader.policy.in.h:3
msgid "Perform a partial upgrade"
msgstr ""

#: ../data/com.ubuntu.release-upgrader.policy.in.h:4
msgid "To perform a partial upgrade, you need to authenticate."
msgstr ""

#: ../do-partial-upgrade:75 ../do-release-upgrade:80
msgid "Show version and exit"
msgstr "Syn versjonen og avslutt"

#: ../do-partial-upgrade:78 ../do-release-upgrade:87
msgid "Directory that contains the data files"
msgstr "Mappe som inneheld datafilene"

#: ../do-partial-upgrade:81 ../do-release-upgrade:100
msgid "Run the specified frontend"
msgstr "Køyr vald grenseflate"

#: ../do-partial-upgrade:112
msgid "Running partial upgrade"
msgstr "Køyrer delvis oppgradering"

#: ../do-release-upgrade:34 ../do-release-upgrade:46
msgid "Downloading the release upgrade tool"
msgstr "Lastar ned oppgraderingverktøyet"

#: ../do-release-upgrade:83
msgid ""
"If using the latest supported release, upgrade to the development release"
msgstr ""

#: ../do-release-upgrade:90 ../check_new_release_gtk.py:181
msgid ""
"Try upgrading to the latest release using the upgrader from $distro-proposed"
msgstr ""
"Forsøk å oppgradera til den nyaste utgjevinga ved å bruka "
"oppgraderingsverktøyet frå $distro-proposed"

#: ../do-release-upgrade:94
msgid ""
"Run in a special upgrade mode.\n"
"Currently 'desktop' for regular upgrades of a desktop system and 'server' "
"for server systems are supported."
msgstr ""
"Køyr i ein spesiell oppgraderingsmodus.\n"
"No vert 'desktop' støtta for oppgradering av vanlege arbeidsstasjonar, og "
"'server' for tenarsystem."

#: ../do-release-upgrade:103
msgid ""
"Check only if a new distribution release is available and report the result "
"via the exit code"
msgstr ""
"Berre sjekk om ei ny distribusjonsutgjeving er tilgjengeleg, og returner "
"resultatet via avsluttingskoden."

#: ../do-release-upgrade:108
msgid ""
"Try the upgrade with third party mirrors and repositories enabled instead of "
"commenting them out."
msgstr ""

#: ../do-release-upgrade:122
msgid "The options --devel-release and --proposed are"
msgstr ""

#: ../do-release-upgrade:123
msgid "mutually exclusive. Please use only one of them."
msgstr ""

#: ../do-release-upgrade:127
msgid "Checking for a new Ubuntu release"
msgstr ""

#: ../do-release-upgrade:139
msgid "Your Ubuntu release is not supported anymore."
msgstr "Di utgåve av Ubuntu er ikkje lenger støtta."

#: ../do-release-upgrade:140
msgid ""
"For upgrade information, please visit:\n"
"%(url)s\n"
msgstr ""
"For informasjon om oppgradering, vitj:\n"
"%(url)s\n"

#: ../do-release-upgrade:147
msgid "In /etc/update-manager/release-upgrades Prompt "
msgstr ""

#: ../do-release-upgrade:148
msgid "is set to never so upgrading is not possible."
msgstr ""

#: ../do-release-upgrade:150
msgid "There is no development version of an LTS available."
msgstr ""

#: ../do-release-upgrade:151
msgid "To upgrade to the latest non-LTS development release "
msgstr ""

#: ../do-release-upgrade:152
msgid "set Prompt=normal in /etc/update-manager/release-upgrades."
msgstr ""

#: ../do-release-upgrade:154
msgid "Upgrades to the development release are only "
msgstr ""

#: ../do-release-upgrade:155
msgid "available from the latest supported release."
msgstr ""

#: ../do-release-upgrade:157
msgid "No new release found."
msgstr ""

#: ../do-release-upgrade:162
msgid "Release upgrade not possible right now"
msgstr "Å oppgradere til ny utgåve er ikkje mogleg akkurat no"

#: ../do-release-upgrade:163
#, c-format
msgid ""
"The release upgrade can not be performed currently, please try again later. "
"The server reported: '%s'"
msgstr ""
"Klarte ikkje å oppgradera til den nye versjonen akkurat no. Prøv på nytt "
"seinare. Tenaren svarte: «%s»"

#: ../do-release-upgrade:169
#, c-format
msgid "New release '%s' available."
msgstr "Ei ny utgjeving, «%s», er tilgjengeleg."

#: ../do-release-upgrade:170
msgid "Run 'do-release-upgrade' to upgrade to it."
msgstr "Køyr «do-release-upgrade» for å oppgradera til henne."

#: ../do-release-upgrade:187
msgid "Please install all available updates for your release before upgrading."
msgstr ""

#: ../do-release-upgrade:203
msgid ""
"You have not rebooted after updating a package which requires a reboot. "
"Please reboot before upgrading."
msgstr ""

#: ../check_new_release_gtk.py:85
msgid "Software Updater"
msgstr ""

#: ../check_new_release_gtk.py:113
#, python-format
msgid "Ubuntu %(version)s Upgrade Available"
msgstr "Oppgradering tilgjengeleg: Ubuntu %(version)s"

#. print("don't upgrade")
#: ../check_new_release_gtk.py:144
#, python-format
msgid "You have declined the upgrade to Ubuntu %s"
msgstr "Du har takka nei til å oppgradera åt Ubuntu %s"

#: ../check_new_release_gtk.py:177
msgid "Check if upgrading to the latest devel release is possible"
msgstr "Sjekk om ei oppgradering til den siste utviklarversjonen er mogeleg"

#: ../check_new_release_gtk.py:186
msgid "Add debug output"
msgstr ""

#~ msgid ""
#~ "An unresolvable problem occurred while calculating the upgrade:\n"
#~ "%s\n"
#~ "\n"
#~ " This can be caused by:\n"
#~ " * Upgrading to a pre-release version of Ubuntu\n"
#~ " * Running the current pre-release version of Ubuntu\n"
#~ " * Unofficial software packages not provided by Ubuntu\n"
#~ "\n"
#~ msgstr ""
#~ "Eit problem som ikkje kan løysast automatisk oppstod medan oppgraderinga "
#~ "vart førebudd:\n"
#~ "%s\n"
#~ "\n"
#~ " Dette kan vera årsakene:\n"
#~ " * Oppgradering til ei tidleg utgåve (før-utgåva) av ein Ubuntu-versjon\n"
#~ " * Bruken av den noverande før-utgavå av ein Ubuntu-versjon\n"
#~ " * Uoffisielle programpakkar som ikkje kjem frå Ubuntu\n"
#~ "\n"

#~ msgid ""
#~ "Your system does not contain a ubuntu-desktop, kubuntu-desktop, xubuntu-"
#~ "desktop or edubuntu-desktop package and it was not possible to detect "
#~ "which version of Ubuntu you are running.\n"
#~ " Please install one of the packages above first using synaptic or apt-get "
#~ "before proceeding."
#~ msgstr ""
#~ "Systemet ditt inneheld ingen av pakkane ubuntu-desktop, kubuntu-desktop "
#~ "eller edubuntu-desktop og det var ikkje mogeleg å finna ut kva ubuntu-"
#~ "versjon du brukar.\n"
#~ "Installer ein av desse pakkane ved å bruka synaptic eller apt-get før du "
#~ "fortset."

#~ msgid "Sandbox setup failed"
#~ msgstr "Sandkasseoppsett var mislukka"

#~ msgid "It was not possible to create the sandbox environment."
#~ msgstr "Klarte ikkje å oppretta sandkassemiljøet."

#~ msgid "Sandbox mode"
#~ msgstr "Sandkassemodus"

#~ msgid ""
#~ "Your python install is corrupted. Please fix the '/usr/bin/python' "
#~ "symlink."
#~ msgstr ""
#~ "Python-installasjonen din er øydelagd. Den symbolske lenkja «/usr/bin/"
#~ "python» må reparerast."

#~ msgid "Package 'debsig-verify' is installed"
#~ msgstr "Pakken «debsig-verify» er installert"

#~ msgid ""
#~ "The upgrade can not continue with that package installed.\n"
#~ "Please remove it with synaptic or 'apt-get remove debsig-verify' first "
#~ "and run the upgrade again."
#~ msgstr ""
#~ "Oppgraderinga kan ikkje halda fram med den pakken installert.\n"
#~ "Fjern han med «synaptic» eller «apt-get remove debsig-verify» fyrst og "
#~ "køyr oppdateringa om att."

#~ msgid ""
#~ "While scanning your repository information no mirror entry for the "
#~ "upgrade was found. This can happen if you run a internal mirror or if the "
#~ "mirror information is out of date.\n"
#~ "\n"
#~ "Do you want to rewrite your 'sources.list' file anyway? If you choose "
#~ "'Yes' here it will update all '%s' to '%s' entries.\n"
#~ "If you select 'No' the upgrade will cancel."
#~ msgstr ""
#~ "Det vart ikkje funne tenarar for oppgradering i arkivinformasjonen din. "
#~ "Dette kan skje om du køyrer ein intern tenar eller viss tenarlista er "
#~ "utdatert.\n"
#~ "\n"
#~ "Vil du skriva om «sources.list»? Om du vel «Ja» her, så vert alle "
#~ "oppføringane frå «%s» til «%s» oppdaterte. Vel du «Nei», vert "
#~ "oppdateringa avbroten."

#~ msgid ""
#~ "The upgrade has aborted. The upgrade needs a total of %s free space on "
#~ "disk '%s'. Please free at least an additional %s of disk space on '%s'. "
#~ "Empty your trash and remove temporary packages of former installations "
#~ "using 'sudo apt-get clean'."
#~ msgstr ""
#~ "Oppgraderinga vart avbroten. Det trengst totalt %s ledig plass på disken "
#~ "«%s». Frigjer minst %s diskplass på «%s». Tøm papirkorga og fjern "
#~ "mellombelse pakkar frå tidlegare installasjonar ved hjelp av «sudo apt-"
#~ "get clean»."

#~ msgid "evms in use"
#~ msgstr "«evms» i bruk"

#~ msgid ""
#~ "Your system uses the 'evms' volume manager in /proc/mounts. The 'evms' "
#~ "software is no longer supported, please switch it off and run the upgrade "
#~ "again when this is done."
#~ msgstr ""
#~ "Systemet ditt brukar «evms» for å handsama dataområde i «/proc/mounts». "
#~ "«evms»-programvaren er ikkje lenger støtta, slå henne av og køyr "
#~ "oppgraderinga på nytt."

#~ msgid ""
#~ "This computer is currently using the NVIDIA 'nvidia' graphics driver. No "
#~ "version of this driver is available that works with your video card in "
#~ "Ubuntu 10.04 LTS.\n"
#~ "\n"
#~ "Do you want to continue?"
#~ msgstr ""
#~ "Datamaskina brukar NVIDIA «nvidia»-grafikkdrivaren. Ingen tilgjengelege "
#~ "versjonar av denne drivaren verkar med skjermkortet ditt i Ubuntu 10.04 "
#~ "LTS. \n"
#~ "\n"
#~ "Vil du halda fram?"

#~ msgid "No i686 CPU"
#~ msgstr "Ikkje ein i686-prosessor"

#~ msgid ""
#~ "Your system uses an i586 CPU or a CPU that does not have the 'cmov' "
#~ "extension. All packages were built with optimizations requiring i686 as "
#~ "the minimal architecture. It is not possible to upgrade your system to a "
#~ "new Ubuntu release with this hardware."
#~ msgstr ""
#~ "Systemet brukar ein i586-prosessor, eller ein prosessor som ikkje har "
#~ "«cmov»-utvidinga. Alle pakkane krev minimum i686-arkitektur. Det er ikkje "
#~ "mogleg å oppgradera systemet ditt til den nyaste Ubuntu-utgjevinga med "
#~ "den noverande maskinvaren."

#~ msgid "Sandbox upgrade using aufs"
#~ msgstr "Sandkasseoppgradering ved bruk av aufs"

#~ msgid "Test upgrade with a sandbox aufs overlay"
#~ msgstr "Test oppgraderinga i ei aufs-sandkasse"

#~ msgid "No new release found"
#~ msgstr "Fann ingen nye utgjevingar"
